### ISSUE TITLE
Fineturning for text classification 

### ISSUE THREAD

**Issue by choudharya3** [cgmhaicenter/exBERT#10]
Hi,
I want to finetune an existing BERT model for classification on a medical domain with some words added in the vocabulary that are important in my training data. However, I am not able to generate the .pkl files for the data ,after following the steps. My concerns are : 
1) Are the steps you mentioned valid for sequence classification data ? As I can see, the objective is Next Sentence Prediction ( NSP ). However, for Language Modelling, are these steps useful ? 
2) I have training data in the form of a dataframe ( text & labels ). How do I go ahead with the task of finetuning using this data ( with additional vocab ). 
        I have already tried adding new words to the 'vocab.txt' file in the model checkpoints, but this did not work out, as the generated model was not recognising the new words I previously added .

Regards,
